hooks/ecosystem-conversion-utils.ts

```diff
@@ -12,6 +12,10 @@ async function fetchMembers() {
     const membersArray: Member[] = [];
     Object.values(res.data).forEach((tier: any) => {
       Object.values(tier).forEach((member: any) => {
+        // convert null to 0
+        if (member.stars === null) {
```

Comment:
Not sure of the value we can get. Right now is `null`, but maybe we can get other `falsy` values, so I think it's better to directly check if there are no stars

```suggestion
        if (!member.stars) {
```

Also, could you explain on the comment that we are doing this to help us with the order?

Reply:
@y4izus, done [here](https://github.com/Qiskit/qiskit.org/pull/3335/commits/f2cbe3aa966c7eafa2e332b6e0e99f6d525d12e2)